packages/fast-components-react-msft/src/subheading/subheading.schema.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json.schema.org/schema#",
+    "title": "Subheading",
+    "description": "A Subheading component's schema definition",
+    "type": "object",
+    "id": "Subheading",
+    "children": true,
+    "properties": {
+        "tag": {

```

Comment:
since the interface specifies that tag is required, add a required field containing "tag" (see toggle.schema.json in the fast-components-react-base) looks like this is missing from some of our schemas but schemas should always match the typescript interface.

Reply:
making tag optional since Nick's comment above but learned something about the schema bits